### ISSUE TITLE
[Enhancement]: drill out "abridged : yes / now" to Item-Type

### ISSUE THREAD

**Issue by Gru0810** [advplyr/audiobookshelf#2482]
### Describe the feature/enhancement

Today I can only mark  a item as "abridged" or not.
But it would be very help full to be able to have  following  types
- Audiobook full reading ( today equal with abridged = no)
- Audiobook abridged reading ( today equal with abridged = yes)
- Radio play
- Live-act ( comedian, cabaret etc)
- (what else ?)
Some items exists as Audiobooks but also as Radio play!
Perhaps you can use the existing field "abridged" ( integer)  in table "books"
-1 = undef ( default for new item)
0 = Audiobook abridged = no
1 = Audiobook abridged = yes
2 = Radio play
3 = Live Act

I have thousands of items and as more items you have as more important is the possibility of an differentiated search.

**Comment by nichwall** [comment]
Can you use a Tag to differentiate the different types?

**Comment by Gru0810** [comment]
yes this would be possible. My workaround today is the usage of Genre 
![image](https://github.com/advplyr/audiobookshelf/assets/72630160/2dc6fb10-1958-4e6e-93fd-729207630e64)
Also Audible differentiate  between Audioboos (full/abr) and Radio play ;)..
and I realy thin to user the existing field ( and rename it would be a easy way, so we can keep the stored data.

**Comment by nichwall** [comment]
Will there need to be combinations of items types? Like can a single item be both "Abridged" and "Radio Play" because there may also both "Unabridged" and "Radio Play"?

**Comment by Gru0810** [comment]
I don't think so .The typeof an item can only be either audiobook or radio play or Live act. Only Audio book can be full or abridged"
